src/main/resources/migrations/master.xml

```diff
@@ -9,5 +9,9 @@
   <include file="migrations/changeset/0003_add_tenant_can_edit_legaltexts_applevel_setting/0003_add_tenant_can_edit_legaltexts_applevel_setting.xml"/>
   <include file="migrations/changeset/0004_add_topic_admin_role_to_super_admin_user/0004_add_topic_admin_role_to_super_admin_user.xml"/>
   <include file="migrations/changeset/0005_add_permission_to_technical_role_rocketchat/0005_add_permission_to_technical_role_rocketchat.xml"/>
+<<<<<<< HEAD
   <include file="migrations/changeset/0006_create_admin_user_table_entries_for_existing_admins/0006_create_admin_user_table_entries_for_existing_admins.xml"/>
+=======
+  <include file="migrations/changeset/0006_change_autoselect_postcode/0006_change_autoselect_postcode.xml"/>
```

Comment:
we don't want to include migration for change autoselect postcode in opensource. It may only be executed for tsys

Reply:
As said for the above issue..